packages/cardano-graphql-services/src/TxSubmit/TxSubmitHttpServer.ts

```diff
@@ -0,0 +1,53 @@
+import { HttpServer, ListenConfig } from '../Http';
+import { Logger, dummyLogger } from 'ts-log';
+import { TxSubmitProvider } from '@cardano-sdk/core';
+import express, { Router } from 'express';
+const bodyParser = require('body-parser');
+
+export interface TxSubmitServerDependencies {
+  txSubmitProvider: TxSubmitProvider;
+  logger?: Logger;
+}
+
+export class TxSubmitHttpServer extends HttpServer {
+  private constructor(config: ListenConfig, router: Router, logger = dummyLogger) {
+    super({ listen: config, name: 'TxSubmitServer', router }, logger);
+  }
+  static create(config: ListenConfig, { txSubmitProvider, logger = dummyLogger }: TxSubmitServerDependencies) {
+    const router = express.Router();
+    // Review: Can we set this limit dynamically?
+    router.use(bodyParser.raw({ limit: '500kB', type: 'application/cbor' }));
+    router.get('/health', async (req, res) => {
+      logger.debug('/health', { ip: req.ip });
+      let body: { ok: boolean } | Error['message'];
+      try {
+        body = await txSubmitProvider.healthCheck();
+      } catch (error) {
+        logger.error(error);
+        body = error.message;
+        // Todo: Inspect errors, and set code based on type of error.
```

Comment:
Could `export txSubmissionErrrors = [BadInputsError, AlreadyDelegatingError, ...]` from `TxSubmissionErrors.ts`, and then do: `if txSubmissionErrors.some(Error => error instanceof Error) then 400`

Reply:
How about this? https://github.com/input-output-hk/cardano-js-sdk/pull/206/commits/0e67ca06564a256ad5f7e9e7a13797b0abf5b74e. It's to enable https://github.com/input-output-hk/cardano-js-sdk/pull/206/commits/ad2f2832b13dcbd6dfbefad0cec522d6dd5c7fe3